EpiAware/src/EpiAware.jl

```diff
@@ -35,16 +35,19 @@ export scan,
     create_discrete_pmf,
     growth_rate_to_reproductive_ratio,
     generate_observation_kernel,
-    default_rw_priors
+    default_rw_priors,
+    neg_MGF,
```

Comment:
Don't have to; its a purely stylistic choice.

Reply:
I'd generally go with only exporting things we want people to use/we will use elsewhere but I don't mind and I don't understand the landscape

In R this wouldn't be a stylistic choice as you are making a statement that you expect users to make use of a given function and therefore wont remove or break it without depreciation notice.

Reply:
> It is common to export names which form part of the API (application programming interface). In the above code, the export list suggests that users should use nice and DOG. However, since qualified names always make identifiers accessible, this is just an option for organizing APIs: unlike other languages, Julia has no facilities for truly hiding module internals.

This makes me think internal functions shouldn't be in the export list. I will make an issue to address.